Safewaan/HonkedIn
# Add descriptions to all APIs, fix some inconsistencies with API naming

## Implementation description
* Added descriptions to APIs in comments to show what their functionality is 
* Renamed `events/getParticipants.js` to `eventParticipants/getEventParticipantsByUserID.js`, as it is filtering the results from the `eventParticipants` table by `userID`
* Renamed `getEventsByUser.js` to `getEventsByUserID.js` as it is filtering the results by `userID`
* Renamed `userSearchByEmail.js` to `getUserByEmail.js` for consistency with our other search APIs naming convention
* Updated all references to the above APIs 



## Steps to test
1. Open the "My Events" page. Verify that your created events appear. This validates that changes to `userSearchByEmail` and `getEventsByUser` have no issues.
2. Click on the "See Participants" button. Verify that the appropriate participants appear. This validates that changes to `getParticipants` have no issues.


## What should reviewers focus on?
* Are there any inconsistencies with naming? Do the updated names fit?
